components/avatar/avatar_default.story.vue

```diff
@@ -4,6 +4,7 @@
     :size="size"
     :presence="presence"
     :initials="initials"
+    :seed="seed"
```

Comment:
nit: add this prop to the icon story too, I think this can be useful to find a good combination of background/icon or for testing purposes.

Reply:
good call, added